README.md

```diff
@@ -453,8 +453,8 @@ The BibTeX entries for these references can be found in the `doc/bib/references.
 
 The following copyright applies to each file in the CEED software suite, unless otherwise stated in the file:
 
-> Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at the
-> Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights reserved.
+> Copyright (c) 2017-2023 Lawrence Livermore National Security, LLC.
```

Comment:
Ratel includes CU and 'other authors'. Should we do that here too?

Reply:
I think this just wasn't updated to match what's in the source files, e.g.,
```c
// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
//
// SPDX-License-Identifier: BSD-2-Clause
```